.idea/GitAssemblyUtil.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What's the test coverage for the diff?

Reply:
deleted.

Reply:
????

Reply:
the .idea folder isn't included anymore.

Reply:
The comment you're replying to is:

> What's the test coverage for the diff?

Reply:
what's the diff?

Reply:
the difference between the old state of the repo and the new state of the repo your PR will create.

You can see this by doing `git diff` or looking at the files tab above.

Reply:
92%

Reply:
Is there a reason why the diff isn't completely covered?

Reply:
The 5 of the 6 lines that aren't covered are for the MGUError log.
file:///Users/cheyennenelson/Documents/GitHub/assemblyUtil/test_local/workdir/test_coverage/AssemblyUtil_TypeToFasta_py.html

Reply:
That seems like something that should be tested if possible

Reply:
It's a try / except to catch the appropriate ServerException log it. How would you unit test for a ServerException? 
In the try and expect mgu example you gave me, it's not tested against for the dfu in https://github.com/kbaseapps/kb_quast/blob/master/test/kb_quast_server_test.py....

Reply:
You'd provide input such that the exception block would be exercised. There are definitely tests in kb_quast that trigger workspace exceptions, for example. Exception handling is one of the places where bugs can be really devastating, based on a SW management class I took a year or two ago, so if they can be tested (which is not always the case) it's good to test them.

for example, you could test a try/catch block for WS by providing invalid refs in the input (say a WS id of `100000000`). What is pretty annoying to test is that the logs are correct (you'd have to redirect stdout/err) so I usually just test that manually.